src/bundle.ts

```diff
@@ -44,6 +39,11 @@ export class Bundler extends Transform {
     this.config = config;
     this.analyzer = analyzer;
     this.sharedBundleUrl = 'shared-bundle.html';
+    this.bundler = new Bundler(<BundlerOptions>{
```

Comment:
These typings might be overkill. Was trying to please annoyance warnings from the tslint. . .